Source/WebKit/Platform/cocoa/ImageAnalysisUtilities.mm

```diff
@@ -193,7 +193,7 @@ static TextRecognitionResult makeTextRecognitionResult(VKCImageAnalysisTranslati
             continue;
         }
 
-        if ([paragraph respondsToSelector:@selector(isPassthrough)] && [paragraph isPassthrough])
+        if (paragraph.isPassthrough)
```

Comment:
This code only applies to macOS 13+, and the `isPassthrough` SPI was added in macOS 12. ✅